lib/execution_engine2/sdk/EE2Runjob.py

```diff
@@ -64,9 +75,11 @@ def _init_job_rec(
         inputs.method = params.get("method")
         inputs.params = params.get("params")
 
-        params["service_ver"] = self._get_module_git_commit(
-            params.get("method"), params.get("service_ver")
-        )
+        if git_commit:
```

Comment:
We are allowing specifying differing versions of apps, which is similar but not exactly as described in https://github.com/kbase/execution_engine2/issues/270